scripts/release.sh

```diff
@@ -136,7 +136,10 @@ function run_validation_tests() {
   if (( ! SKIP_TESTS )); then
     banner "Running release validation tests"
     # Run tests.
-    $1
+    if ! $1; then
+      banner "Release validation tests failed, aborting"
+      exit 1
```

Comment:
Is this additional exit necessary? Does it clobber the exit status of the test command?

Reply:
It's not additional, and it ensures that the flow stops here. Technically it does clobber the status of the test command, but right now we're not using the status code to take any actions (except signaling a failure).